source/introduction.rst

```diff
@@ -0,0 +1,90 @@
+.. SPDX-FileCopyrightText: 2020 Hyperledger
+   SPDX-License-Identifier: CC-BY-4.0
+
+   perun-doc documentation master file, created by
+   sphinx-quickstart on Thu May 17 17:20:50 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+
+Introduction
+=============
+
+Perun
+-----
+
+Perun is an off-chain framework which allows users to make transactions without interacting with the blockchain. The main objective of the project is to make the blockchains ready for mass adoption and alleviate current technical challenges such as high fees, latency and low transaction throughput. Perun protocol can be used on top of any blockchain system to accelerate decentralized applications and lower transaction fees.
+
+The perun protocol
+------------------
+
+The Perun protocol allows users to shift transaction and smart contract execution away from the blockchain into so-called payment and state-channels. These channels are created by locking coins on the blockchain and can be updated directly between the users and without any on-chain interaction. This makes state-channel-based transactions much faster and cheaper than on-chain transactions. The underlying blockchain guarantees that all off-chain transactions will be enforced on-chain eventually. In comparison to other channel technologies like the Lightning Network, the Perun construction offers the following unique features:
+
+Perun’s state-channel virtualization
+````````````````````````````````````
+To connect users that do not have a joint open state-channel, existing state-channels can be composed to form so-called virtual channels. These virtual channels are created and closed off-chain over the state-channel network intermediaries. Once opened, the virtual channel is updated directly off-chain between the two connected end users.
```

Comment:
Not sure if we should highlight this is as the first point because it's not implemented yet. Or add a remark that the feature is still to be implemented?